### DISCUSSION TITLE
What error number 0 means?

### DISCUSSION THREAD

**Discussion by AlfaGTA** [Mimetis/Dotmim.Sync#785] [Q&A]
In some cases the synchronization process returns an error with number code 0. I cannot understand what the cause of the problem is, because the description of the message in some cases is different. For example, in one case it was "Unterminated string. Expected delimiter: ". Path 'changes.t[0].r[828][0]', line 1, position 142595, while another time was "Network is unreachable". 
So, what does error 0 refer to? Is the origin always the same? If so, which one (Network is unreachable i think)? Otherwise, how can I distinguish the reasons that cause it?

**Comment by Mimetis** [comment]
Honestly no idea ....

Do you have any idea why this "`Unterminated string. Expected delimiter: ". Path 'changes.t[0].r[828][0]', line 1, position 142595`" is happening ?
Are you able to reproduce it ?

**Comment by AlfaGTA** [comment]
It is not so easy to reproduce. I believe it is due to a sudden loss of the data connection during synchronization, but it is just a suspicion.
I try to get the hresult of the error and let you know.
